doc/releases/changelog-dev.md

```diff
@@ -145,6 +148,9 @@
 
 <h4>Community contributions 🥳</h4>
 
+* Resolved the bug in `qml.ThermalRelaxationError` where there was a typo from `tq` to `tg`.
+  [(#5988)](https://github.com/PennyLaneAI/pennylane/issues/5988)
+
```

Comment:
I'm not sure. It may be a merge conflict, I'll fix it.

Reply:
This was because the entry for this bug fix was originally under "New features since last release", which is incorrect. So I moved it to "Community Contributions" as it was an external contributor's PR.